assets/javascripts/src/components/Ticker.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+
+function toK(val) {
+    return val / 1000;
+}
+
+function positionTarget(props) {
+    return (props.target / props.overfund * 100) + '%';
+}
+
+function percentageAsNegativeTranslate(props) {
+    return Math.floor(props.total / props.overfund * 100 - 100) + '%';
+}
+
+export default class Ticker extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {count: 0, filledProgress: '-100%'};
+    }
+
+    render() {
+        return (
+            <div className='ticker'>
+                <div className="ticker__wrapper l-constrained">
+                    <h3 className="ticker__count">${this.state.count.toLocaleString()}</h3>
+                    <h4 className="ticker__label">pledged so far</h4>
+                    <div className="ticker__progress">
+                        <div className="ticker__filled-progress" style={{transform: 'translateX(' + this.state.filledProgress + ')'}}></div>
```

Comment:
I would separate out the progress bar and the counter to be sub components. Then rather than having to re-render the entire html block every `30ms` we would only be updating the progress bar and counter elements, which would be more performant.